diff --git a/src/run_mopac.F90 b/src/run_mopac.F90
index d502cfc..75ddceb 100644
--- a/src/run_mopac.F90
+++ b/src/run_mopac.F90
@@ -104,12 +104,15 @@
 #endif
 ! parse command-line flags
 #ifdef MOPAC_F2003
+write(*,*) "COMMAND LINE ARGUMENT NUMBER:", command_argument_count()
       do i = 1, command_argument_count()
         call get_command_argument (i, jobnam)
 #else
+  write(*,*) "COMMAND LINE ARGUMENT NUMBER (old):", iargc()
       do i = 1, iargc()
         call getarg (i, jobnam)
 #endif
+write(*,*) "COMMAND LINE ARGUMENTS:", i, jobnam
         if (jobnam == '-V' .OR. jobnam == '--version') then
 #ifdef MOPAC_GIT_HASH
           write(*,"(a)") "MOPAC version "//trim(verson)//" commit "//trim(git_hash)
